package.json

```diff
@@ -75,19 +91,25 @@
         "eslint-plugin-react": "^7.19.0",
         "file-loader": "^6.2.0",
         "indefinitely-typed": "^1.1.0",
+        "jest": "^26.6.3",
         "json-schema": "^0.3.0",
+        "jsverify": "^0.8.4",
         "mini-css-extract-plugin": "^1.3.9",
+        "optimize-css-assets-webpack-plugin": "^5.0.3",
         "prettier": "^2.2.1",
         "R": "^1.0.0",
         "raw-loader": "^4.0.2",
         "react-ace": "^9.3.0",
         "react-docgen": "^5.3.1",
-        "source-map-loader": "^2.0.1",
+        "react-resize-panel": "^0.3.5",
+        "react-test-renderer": "^17.0.1",
+        "redux-mock-store": "^1.5.4",
+        "source-map-loader": "^0.2.4",
         "typescript": "^4.2.3",
         "url-loader": "^4.1.1",
-        "webpack": "^5.28.0",
-        "webpack-cli": "^4.5.0",
-        "webpack-dev-server": "^3.11.2"
+        "webpack": "^4.41.2",
```

Comment:
Maybe create an issue to switch back to webpack5, when Storybook properly supports it.